####################### PDDL requirements
string domain_name

# for Domain.pddl
std_msgs/String[] types
std_msgs/String[] constants
std_msgs/String[] predicates
std_msgs/String[] actions 

# for Problem.pddl
std_msgs/String[] problem_name                      
std_msgs/String[] objects
std_msgs/String[] inits
std_msgs/String[] goals
